### PR TITLE
(maint) Use GH Actions deployment workflow

### PR DESCRIPTION
**Pull request by glennsarti**
Previously Travis CI has stopped working the docs project and we now have to use
GitHub Actions.  This commit migrates the Travis deployment process to use
GH Actions.

Note - The GITHUB_TOKEN secret needs to be setup in the GH Repository

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        with:
+          node-version: '12.x'
+
+      - name: Cache dependencies
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
```
Comment (jpogran) [inline .github/workflows/deploy.yml:40]: did you mean to have a trailing dash here?

Comment (glennsarti) [reply .github/workflows/deploy.yml:40]: Honestly...nope :-)

Comment (glennsarti) [reply .github/workflows/deploy.yml:40]: Actually...maybe...that is correct

Comment (glennsarti) [reply .github/workflows/deploy.yml:40]: ahh ok ... it's a prefix .... so yes trailing dash is fine

https://docs.github.com/en/free-pro-team@latest/actions/guides/caching-dependencies-to-speed-up-workflows#input-parameters-for-the-cache-action

**Review by jpogran — APPROVED** [review]

**Pull request merged**